website/content/docs/k8s/deployment-configurations/external-service.mdx

```diff
@@ -22,7 +22,9 @@ The process to register an external service in Consul on Kubernetes consists of
 1. [Start Consul ESM](/consul/tutorials/connect-services/service-registration-external-services#monitor-the-external-service-with-consul-esm). You must use Consul ESM to run health checks on external services.
 1. Define the external service and its health checks in a [`Registration` CRD](/consul/docs/connect/config-entries/registration).
 1. Apply the CRD to your Kubernetes cluster. Internally, this action triggers an API call to Consul's [`/catalog/register` endpoint](/consul/api-docs/catalog#register-entity) to register the service.
-1. When using Consul's service mesh, define [service intentions](/consul/docs/connect/intentions) between the external service and services it communicates with.
+1. When using Consul's service mesh, you should also:
+  - Deploy a [terminating gateway](/consul/docs/k8s/connect/terminating-gateways) so that the external service can communicate with downstream services.
```

Comment:
```suggestion
  - Deploy a [terminating gateway](/consul/docs/k8s/connect/terminating-gateways) so that downstream services can communicate with the external service.
```